app/models/fine.rb

```diff
@@ -38,10 +38,16 @@ class Fine
     EQUPREPAIR: 'Fee for repair costs of equipment damaged by borrowers'
   }.with_indifferent_access.freeze
 
+  NO_TITLE_TEXT = 'NOT ASSOCIATED WITH AN ITEM'
+
   def initialize(record)
     @record = record
   end
 
+  def title
+    bib['title'] || NO_TITLE_TEXT
```

Comment:
Is `"block"=>{"resource"=>"/policy/block", "key"=>"MISC"},` more reliable? Or, not having a title is just as good right?

Reply:
The rule they use in current my account is for bills when there is no title so I tried to mimic that.

Reply:
okie dokie